### PR TITLE
WIP Rename files PART 4

### PR DESCRIPTION
**Pull request by smarletta**

### CODE REVIEW

File: tests/unit/parsecmdline.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const assert = require('chai').assert;
 if (!global.window)
   global.window = new Object({});
 
-require('../../saltgui/static/scripts/parsecmdline');
+require('../../saltgui/static/scripts/ParseCommandLine');
 
 
 describe('Unittests for parsecmdline.js', function() {
```
Comment (erwindon) [inline tests/unit/parsecmdline.js]: shouldn't the unit test have the same filename as the file-under-test?

**Comment by smarletta** [comment]
You are right. Forgot about the tests... Will create a PR

Erwin Dondorp <notifications@github.com> schrieb am Mo., 4. März 2019,
21:55:

**Comment by erwindon** [comment]
> You are right. Forgot about the tests... Will create a PR

why not just update this PR?

**Comment by smarletta** [comment]
You are right, will do

Erwin Dondorp <notifications@github.com> schrieb am Mo., 4. März 2019,
22:05:

**Comment by smarletta** [comment]
Still something missing?

**Comment by erwindon** [comment]
taking the lead for this PR

**Comment by erwindon** [comment]
rebased into origin/renameFiles

**Comment by erwindon** [comment]
merged that branch unchanged

**Pull request closed without merging**